spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,4 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
+
```

Comment:
I think this file was generated automatically, however we should either [implement the preview method](https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails) or delete the file. I'm okay with both, what do you think?